edgelet/signal-future/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "signal-future"
+version = "0.1.0"
+authors = ["Rajasekharan Vengalil <rajave@microsoft.com>"]
```

Comment:
As much as I'd like to take credit, this probably needs to be:
```toml
authors = ["Azure IoT Edge Devs"]
```

Reply:
I was going to give you the credit :)